image.js

```diff
@@ -1,8 +1,17 @@
 const PNGReader = require('png.js');
+const common = require('./common.js');
+
+function convertPNG(pngBytes, rotated, color_depth) {
```

Comment:
As I mentioned in my other comment, I think using flags to differentiate between configurations might get tricky as we add more devices. Let's just stick to exposing the individual methods for now.

- [ ] Expose each method individually

Reply:
Done, please check 69de6b0